### PR TITLE
Feature/mat 169 tech spike - just some UML diagrams

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: uml/convert_current_state.puml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+MicroService --> MAT: report
+MAT --> User: [not valid] return, redirect to show report
+MAT -> MAT: [valid] proceed with conversion
+MAT -> MAT: clone source measure \ninto new draft measure \nwith FHIR type
+MAT -> "MAT DB": persist draft FHIR measure
+"MAT DB" --> MAT: fhirMeasureId
+MAT -> MicroService: convert\n(measureId, type=CONVERSION, format=SIMPLE)
+MicroService -> MicroService: validate
+MicroService -> VSAC: value set validation
+MicroService -> "MicroService's MongoDB": store report
```
Comment (RohitKandimalla) [inline uml/convert_current_state.puml]: We are already Validating and storing report in MongoDB right? After Cloning & validating, will it be stored as a new report ?

Comment (serhii-ilin) [reply uml/convert_current_state.puml]: It was not the intention to provide a technical decision or a future design. I just documented some of the flows in PlantUML format.
You can render sequence diagrams out of that with a PlantUML plugin (Intellij or Confluence).

Comment (serhii-ilin) [reply uml/convert_current_state.puml]: as far as I know, the report is already stored in MongoDB on the micro-service end.

Comment (RohitKandimalla) [reply uml/convert_current_state.puml]: Yup I did get PlantUML

**Review by RohitKandimalla — APPROVED** [review]

**Review by adongare — APPROVED** [review]

**Pull request merged**